### ISSUE TITLE
Immediate File Access

### ISSUE THREAD

**Issue by pkelleter** [pkelleter/angular-cli-quick-switch-plugin#12]
Regarding Daniel Mattsson's comment from 
https://plugins.jetbrains.com/plugin/10587-angular-cli-quickswitch 
I will put his request into a github issue for him.

> I like this plugin a lot. One thing that would make it even better for is if I could assign a shortcut for each file type. E.g. Ctrl + Shift + H would always take me to the html file, Ctrl + Shift + S to the .scss file etc.

**Comment by kkorus** [comment]
Yes, please. Like this VSCode plugin https://marketplace.visualstudio.com/items?itemName=infinity1207.angular2-switcher
`alt + u` Typescript
`alt + i`  Css\scss
`alt + o`  Html
`alt + p` Spec file

**Comment by pkelleter** [comment]
Hey @kkorus 

It took a while but thanks to agentic coding I finally have time again to work on my old plugin 😅

Version 2.0 adds dedicated, reassignable shortcuts for each file category:

Alt+X: controllers
Alt+C: styles
Alt+V: markup
Alt+T: tests

Each shortcut jumps directly to the first existing file in its category and cycles within that category when pressed repeatedly. All shortcuts can be reassigned under the IDE’s Keymap settings.

**Comment by kkorus** [comment]
@pkelleter oh, that's so nice 😍 thanks!
